pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -277,11 +281,8 @@ func (s *Service) sendEvent(msg dispatcher.DIDCommMsg, threadID string, nextStat
 	}
 
 	// trigger the registered action events
-	s.lock.RLock()
-	event := s.actionEvent
-	s.lock.RUnlock()
-	event <- didCommEvent
-
+	// TODO: it can be panic if the channel was closed
```

Comment:
Is there an action item (issue) associated with this TODO?

Reply:
no, it is more for general understanding, need to decide whether it is our or a client responsibility

Reply:
Then I would remove the TODO.